docs/snippets/ov_multi.py

```diff
@@ -2,28 +2,53 @@
 from openvino.runtime import Core
 model_path = "/openvino_CI_CD/result/install_pkg/tests/test_model_zoo/core/models/ir/add_abc.xml"
 path_to_model = "/openvino_CI_CD/result/install_pkg/tests/test_model_zoo/core/models/ir/add_abc.xml"
-def Option_1():
-#! [Option_1]
+def MULTI_0():
+#! [MULTI_0]
     core = Core()
 
-    # Read a network in IR or ONNX format
+    # Read a network in IR, PaddlePaddle, or ONNX format:
     model = core.read_model(model_path)
+    
+    # compile a model on MULTI using the default list of device candidates.
+    compiled_model = core.compile_model(model=model, device_name="MULTI")
+
+    # Optional
+    # You can also specify the devices to be used by MULTI.
+    # The following lines are equivalent:
     compiled_model = core.compile_model(model=model, device_name="MULTI:CPU,GPU")
-#! [Option_1]
+    compiled_model = core.compile_model(model=model, device_name="MULTI", config={"MULTI_DEVICE_PRIORITIES": "GPU,CPU"})
 
-def Option_2():
-#! [Option_2]
-    core = Core()
+    # Optional
+    # MULTI is pre-configured (globally) with the explicit option.
+    
+    core.set_property(device_name="AUTO", properties={"MULTI_DEVICE_PRIORITIES":"GPU,CPU"})
```

Comment:
this is multi code snippet, no "AUTO" should be involved

Reply:
well spotted, thanks!